test/resources/shipment.js

```diff
@@ -241,6 +241,42 @@ describe('Shipment Resource', () => {
     });
   });
 
+  describe('retrieving smartrates', () => {
```

Comment:
# Reviewer Note

These tests don't really do much for us but I followed the same pattern as other tests.

I also end-to-end tested to ensure we get back the expected results when requesting smartrates and that things like creating a shipment continued to behave as expected.